### DISCUSSION TITLE
Strange Update Issue on Company Devices

### DISCUSSION THREAD

**Discussion by itsredhood** [mozilla/policy-templates#1159] [Q&A]
Dear all,

i joined a company and they asked me about an issue they have for several ?years? now.

All Machines are bound to an Domain from the company.

On all Devices, Firefox is used as primary browser.

Some Machines do the Update without asking for Admin Credentials, some machines need Admin Credentials to install the update.

My thought was a security group in the Active Directory that controls the User Account Control but nobody could answer that from the previous IT.

Otherwise i found a Registry Edit that should get rid of it but it did not work out.

Does anyone of you know this issue?

Best regards

**Comment by mkaply** [comment]
This sounds like the maintenance service doesn't have permission to do the updates on some of those machines.

Is there anything different about the way they are installed on those machines?
